sycl/test-e2e/InvokeSimd/Regression/nbarrier_loop.cpp

```diff
@@ -0,0 +1,212 @@
+// TODO: enable on Windows once driver is ready
+// NOTE: named barrier supported only since PVC
+// REQUIRES: gpu-intel-pvc && linux
+// UNSUPPORTED: cuda || hip
```

Comment:
If https://github.com/intel/llvm/pull/9417 would get merged before this PR, this line won't be necessary.